internal/ingress/annotations/rewrite/main.go

```diff
@@ -85,6 +87,7 @@ func (a rewrite) Parse(ing *networking.Ingress) (interface{}, error) {
 
 	config.Target, _ = parser.GetStringAnnotation("rewrite-target", ing)
 	config.SSLRedirect, err = parser.GetBoolAnnotation("ssl-redirect", ing)
+	config.PreserveTrailingSlash, err = parser.GetBoolAnnotation("preserve-trailing-slash", ing)
```

Comment:
Should check the `err` from the `ssl-redirect` annotation parsing firstly, considering the following case:

* `ssl-redirect` annotation is invalid
* `preserve-trailing-slash` is valid

so `err` will be reset to `nil` and the error process was dismissed.

Reply:
fixed, thanks.